packages/lesmis/src/components/DashboardExportModal/components/DashboardNameSubHeader.js

```diff
@@ -1,16 +0,0 @@
-import React from 'react';
-import { Svg, Line, Text } from '@react-pdf/renderer';
-
-const DashboardNameSubHeader = () => {
-  const dashboardName = 'Schools';
-  return (
-    <Svg width="497" height="3" viewBox="0 0 497 3">
-      <Text x="0" y="15">
-        {dashboardName}
-      </Text>
-      <Line x1="0" y1="20" x2="497" y2="20" stroke="#D13333" strokeWidth="3" />
-    </Svg>
-  );
-};
-
-export default DashboardNameSubHeader;
```

Comment:
This file wasn't being used anywhere yeah?

Reply:
No it doesn't get used.